activestorage/test/models/variant_test.rb

```diff
@@ -166,4 +166,14 @@ class ActiveStorage::VariantTest < ActiveSupport::TestCase
   ensure
     ActiveStorage.variant_processor = :mini_magick
   end
+
+  test "resized variation of TIFF blob" do
+    blob = create_file_blob(filename: "image.tif")
+    variant = blob.variant(resize: "50x50").processed
+    assert_match(/image\.tif/, variant.service_url)
+
+    image = read_image(variant)
+    assert_equal 36, image.width
+    assert_equal 50, image.height
+  end
```

Comment:
Can you move this up under the `resized variation of ICO blob` test?

Reply:
@georgeclaghorn done